.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
+  - todo
```

Comment:
This disables warnings for TODO. We should be able to add TODOs in code with no warnings for this. Enabling this rule will motivate developers to avoid TODOs, which is something useful. What do you think?